man/ocitools-generate.1.md

```diff
@@ -234,6 +234,45 @@ inside of the container.
   The special *PATH* `host` removes any existing UTS namespace from
   the configuration.
 
+**--cpu-shares**=CPUSHARES
```

Comment:
It may be worth using `--linux-cpu-shares` here to leave room for other platforms with their own interpretation of CPU shares.  That will help avoid confusion when someone PRs [`--solaris-ncpus`][1], etc.  It's possible that there is common ground here and we could have a cross-platform `--cpu-shares` option, but if that's the case I'd rather have runtime-spec leading the push towards unification.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-solaris.md#cappedcpu